src/longopt/gmtselect_inc.h

```diff
@@ -24,36 +24,26 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 	/* separator, short_option, long_option,
 	          short_directives,    long_directives,
 	          short_modifiers,     long_modifiers */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'L', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'I', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'N', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'Z', "",
-	          "",                  "",
-	          "",                  "" },
+	{ 0, 'A', "min_area|area",
+	          "",                  "",
+	          "a,l,r,p",           "antarctica,lakes,river_lakes,percent" },
+	{ 0, 'C', "distance|dist2pt",
+	          "",                  "",
+	          "d",                 "distance" },
+	{ 0, 'D', "resolution",
+	          "f,h,i,l,c",         "full,high,intermediate,low,crude",
+	          "f",                 "lower" },
+	{ 0, 'E', "boundary",          "", "", "", "" },
+	{ 0, 'F', "polygon",           "", "", "", "" },
+	{ 0, 'G', "grid_mask",         "", "", "", "" },
+	{ 0, 'I', "reverse|invert",    "", "", "", "" },
+	{ 0, 'L', "dist2line",
+	          "",                  "",
+	          "d,p",               "distance,project" },
+	{ 0, 'N', "feature_mask",      "", "", "", "" },
```

Comment:
I don't like the `feature_mask`. It tells me nothing when I read it. Much prefer to keep it just `mask`. We don't have to be consistent with `gridmask` (that I also prefer to `grid_mask`). We have had another case like this but don't remember which right now.

Reply:
OK, agreed, doing mask|mask_values